src/ucp/wireup/address.c

```diff
@@ -581,7 +582,7 @@ static ucs_status_t ucp_address_do_pack(ucp_worker_h worker, ucp_ep_h ep,
                       UCT_TL_RESOURCE_DESC_ARG(&context->tl_rscs[i].tl_rsc),
                       md_flags, iface_attr->cap.flags,
                       iface_attr->bandwidth.dedicated,
-                      iface_attr->bandwidth.shared,
+                      iface_attr->bandwidth.shared / worker->context->config.est_num_ppn,
```

Comment:
pls fix the **format string**, not the passed value

Reply:
fixed